modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -157,6 +194,7 @@ public function build() {
       '#count_text' => $this->formatPlural($count, '@count item', '@count items'),
       '#url' => Url::fromRoute('commerce_cart.page')->toString(),
       '#content' => $cart_views,
+      '#cart_items' => $cart_items,
```

Comment:
This doesn't match the approach we took for the order summary twig template. Makes more sense to pass the full #order_entity.

Reply:
Thanks for the feedback. I'll need to investigate how this should work out.

Reply:
If i need to pass the full #order_entity then how will this work out when you have multiple order types?

Reply:
If we're using Twig, shouldn't we just skip passing View objects to the template?

One issue we have here is that we have one table for all items. We need to keep `cart_items`, but perhaps it contains an array of order items, keyed by the order ID.

Reply:
EDIT: I see how we fix this. Right now the existing cart block is being harnessed. We need a new theme hook. Maybe `commerce_cart_order_items_table`.

Which has

```
+        <table>
 +          <tbody>
 +          {% for cart_item in cart_items %}
 +            <tr>
 +              <td>{{ cart_item.getQuantity|number_format }} x</td>
 +              <td>{{ cart_item.label }}</td>
 +              <td>{{ cart_item.getTotalPrice|commerce_price_format }}</td>
 +            </tr>
 +          {% endfor %}
 +          </tbody>
 +        </table>
```

Then the `#content` value is an array of those.  The new hook received `#order_entity`